cfgov/jinja2/v1/_includes/molecules/image-inset.html

```diff
@@ -0,0 +1,74 @@
+{# ==========================================================================
+
+    Image Inset
+
+    ==========================================================================
+
+    Description:
+
+    Creates image inset markup when given:
+
+    value:                   Object defined from a StreamField block.
+
+    value.heading:           The heading of the image inset.
+
+    value.image:             A basic image object to insert.
+
+    value.image.upload:      The image object.
+
+    value.image.alt:         String for alt text for screenreaders.
+
+    value.paragraph:         String for text below the image.
+
+    value.is_image_left:     Boolean indicating whether to float the image
+                             to the left.
+
+    value.image_width:       String representing the image width.
+                             Possible values: (150, 270, 300).
+                             Defaults to 270.
+
+   ========================================================================== #}
+
+{# Image sizes are doubled for retina displays. Images created via Wagtail
+   are never stretched or upscaled and maintain their aspect ratio. #}
+{% set image_inset = image(value.image.upload, 'width-' ~ value.image_width
+                     | int * 2) %}
+{% set image_mobile = image(value.image.upload, 'width-1200') %}
+<div class="m-inset
+            m-inset__image
+            {{ 'm-inset__image__left' if value.is_image_left else '' }}
+            {{ 'm-inset__image__' ~ ( value.image_width
+               if value.image_width else '270' ) }}
+            {{ 'm-inset__image__portrait'
```

Comment:
This is necessary to maintain the proper crop aspect ratio. It stems from https://github.com/cfpb/cfgov-refresh/blob/83c1281bd92868437c5a8b611d686f020d53f2dc/cfgov/unprocessed/css/misc.less#L175.